### PR TITLE
update the instruction for installing from operatorhub

### PR DESCRIPTION
**Pull request by mikeshng**
Signed-off-by: Mike Ng <ming@redhat.com>

FYI @berenss the `## Install community operator from OperatorHub.io` change will update both the link and the header. Thanks.

Scott wants to make the following changes:
image

### CODE REVIEW

**Comment by berenss** [comment]
thx Mike!

**Comment by swopebe** [comment]
@mikeshng you can remove the others from review, I will take this one.

File: content/getting-started/integration/app-lifecycle.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ export MANAGED_CLUSTER_NAME=<managed cluster name> # export MANAGED_CLUSTER_NAME
 make deploy-community-managed # make deploy-community-managed GO_REQUIRED_MIN_VERSION:= # if you see warnings regarding go version
 ```
 
-## Install from OperatorHub
-If you are using OpenShift or have `OLM` installed in your cluster, you are able to install the multicluster subscription operator with a released version from the OperatorHub. Details can be found [here](https://operatorhub.io/operator/multicluster-operators-subscription).
+## Install community operator from OperatorHub.io
+If you are using OKD, OpenShift or have `OLM` installed in your cluster, you are able to install the multicluster subscription community operator with a released version from the [OperatorHub.io](https://operatorhub.io). Details can be found [here](https://operatorhub.io/operator/multicluster-operators-subscription).
```
Comment (swopebe) [inline content/getting-started/integration/app-lifecycle.md]: you are able to "you can"

Comment (swopebe) [reply content/getting-started/integration/app-lifecycle.md]: comma before "or have OLM"

**Review by swopebe — APPROVED** [review]

**Pull request merged**